BE/prisma/fixtures/projects/web/2.md

```
@@ -0,0 +1,180 @@
+
+## 팀원 소개
+
+<br/>
+
+<div align="center">
+
+<table>
+  <tr>
+    <td align="center">
+      <b>Danny</b><br/>
+      <img src="https://github.com/HoonDongKang.png?size=120" width="150" height="150" style="border-radius:50%"><br/>
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)